Classes/Command/ReindexCommand.php

```diff
@@ -127,6 +133,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             exit(1);
         }
 
+        if (!empty($input->getOption('owner'))) {
+            if (MathUtility::canBeInterpretedAsInteger($input->getOption('owner'))) {
```

Comment:
Please add 

`use TYPO3\CMS\Core\Utility\MathUtility;`

It's not part of ReindexCommand.php, yet.

Reply:
Done!